samtranslator/model/stepfunctions/events.py

```diff
@@ -173,6 +179,7 @@ class CloudWatchEvent(EventSource):
         "DeadLetterConfig": PropertyType(False, is_type(dict)),
         "RetryPolicy": PropertyType(False, is_type(dict)),
         "State": PropertyType(False, is_str()),
+        "Target": PropertyType(False, is_type(dict)),
```

Comment:
In this CloudWatchEvent [docs](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-property-statemachine-statemachinecloudwatchevent.html) it says that "AWS SAM will add support for new features only to EventBridgeRule". Does that apply here?

Reply:
Hmm I didn't notice this comment. `CloudWatchEvent` and `EventBridgeRule` basically share the same code and it doesn't match what doc says... And this is more like a bug fix instead of a new feature. So I think it's fine to keep it here.

Reply:
If we're pushing customers to use `EventBridgeRule` over `CloudWatchEvent` then I think not adding Target to `CloudWatchEvent` might be a good thing as it gives them another reason to not use it.